tests/unit/test_config.py

```diff
@@ -1327,16 +1300,19 @@ def test_implicit_tasks(
     """
     # Setup
     id_ = 'rincewind'
-    flow_file: 'Path' = tmp_flow_config(id_, f"""
-    [scheduler]
-        {
-            f'allow implicit tasks = {allow_implicit_tasks}'
-            if allow_implicit_tasks is not None else ''
-        }
-    [scheduling]
-        [[graph]]
-            R1 = foo
-    """)
+    flow_file: 'Path' = tmp_flow_config(
+        id_,
+        dedent(f"""
+            [scheduler]
+                {
+                    f'allow implicit tasks = {allow_implicit_tasks}'
+                    if allow_implicit_tasks is not None else ''
+                }
+            [scheduling]
+                [[graph]]
+                    R1 = foo
+        """)
+    )
```

Comment:
This change seems to be causing a flake8 fail in CI, but not for me locally 🤔

Reply:
Try matching the Python version used by the lint job in GH actions?

Reply:
That did it.